src/main/java/uk/gov/hmcts/reform/civil/service/robotics/mapper/EventHistoryMapper.java

```diff
@@ -1937,7 +1942,17 @@ private void buildRespondentPartAdmission(EventHistory.EventHistoryBuilder build
                                                   .miscText(miscText)
                                                   .build())
                                 .build());
+            directionsQuestionnaireFiledEvents.add(
+                buildDirectionsQuestionnaireFiledEvent(builder, caseData,
+                                                       respondent1ResponseDate,
+                                                       RESPONDENT_ID,
+                                                       respondent1DQ,
+                                                       respondent1,
+                                                       true
+                ));
             if (defendant1v2SameSolicitorSameResponse.test(caseData)) {
+                Party respondent2 = caseData.getRespondent2();
```

Comment:
these changes will apply not jut to lr v lip but also lr v lr. is this what we want?

Reply:
yes.. correct applicable for both.